R/process-policy.R

```diff
@@ -24,6 +25,7 @@ find_weights <- function(policy){
 }
 #'
 #'@importFrom purrr map map_dbl
+#'@keywords internal
```

Comment:
This is a small thing, but now is the time to address it: please go through your Roxygen2 documentation and use consistent styling of the delimiter followed by a space: `#' @keywords etc`.